src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.DbRow.cs

```diff
@@ -9,6 +9,9 @@ namespace System.Text.Json
 {
     public sealed partial class JsonDocument
     {
+        /// <summary>
+        ///   This is an implementation detail and MUST NOT be used by source-package consumers.
```

Comment:
Nit: can you remove the extra two-space indent from the beginning of all of the text?

Reply:
I could, I suppose :smile:, but I think it makes it a lot more legible.

Reply:
In particular it's just XML-styled indentation to help identify that a (n e.g. &lt;paramref /&gt;) isn't a new tag, but just still part of the existing tag.

If we have a style thing for it, or there's a tooling problem, I'll certainly go along with it.

Reply:
I just don't remember seeing this style elsewhere in the repo.  Not a big deal, other than I like consistency.  Up to you.

Reply:
FWIW, this spacing doesn't affect how the comments show up in IntelliSense (though the change does show up in the underlying xml file).